screenshots/screenshots.e2e.spec.ts

```diff
@@ -47,9 +47,23 @@ test.describe.serial('Main App Test', () => {
     await page.emulateMedia({ colorScheme });
 
     await navPage.progressBecomesReady();
+
+    await page.waitForTimeout(2500);
+
+    // await tool('rdctl', 'extension', 'install', 'ghcr.io/rancher-sandbox/epinio-desktop-extension:0.0.12');
```

Comment:
Shouldn't have commented-out code in a PR

Reply:
I just left these in for posterity because the intent is to automate the install/uninstall of these extensions during the screenshots task. 

I can just remove these lines and add a comment that they need to be re-added for #4495 instead.

Reply:
Left a reference for us to reintegrate the tool install of expected extensions in https://github.com/rancher-sandbox/rancher-desktop/issues/4495#issuecomment-1513948618